app/Console/Commands/Importer.php

```diff
@@ -7,7 +7,7 @@
 
 class Importer extends BaseCommand
 {
-    protected $signature = 'phpvms:importer {db_host} {db_name} {db_user} {db_pass?}';
```

Comment:
Not sure where my comment went, change this one back for now, I'll probably change this to use a config file, since I think there will be more settings to add. For testing, though, it's a pain with the `ask()` every time